quickjs.c

```diff
@@ -5783,6 +5803,22 @@ void JS_MarkValue(JSRuntime *rt, JSValueConst val, JS_MarkFunc *mark_func)
     }
 }
 
+static void mark_weak_map_value(JSRuntime *rt, JSWeakRefRecord **first_weak_ref, JS_MarkFunc *mark_func) {
```

Comment:
```suggestion
static void mark_weak_map_value(JSRuntime *rt, JSWeakRefRecord *first_weak_ref, JS_MarkFunc *mark_func) {
```
Asking for a pointer-to-pointer suggests the pointed-to value is going to get updated.